sqlitecluster/SQLiteCommand.cpp

```diff
@@ -5,9 +5,7 @@
 
 SData SQLiteCommand::preprocessRequest(SData&& request) {
     // If the request doesn't specify an execution time, default to right now.
-    if (!request.isSet("commandExecuteTime")) {
-        request["commandExecuteTime"] = to_string(STimeNow());
-    } else {
+    if (request.isSet("commandExecuteTime")) {
```

Comment:
Would it be worth setting this to some other key if it's not already set? Or does that cause problems?

Reply:
Discussed with Tyler, sounds like it's not needed if the command is not intended to be run in the future